subsys/random/rand32_xoshiro128.c

```diff
@@ -41,9 +41,9 @@ static inline uint32_t rotl(const uint32_t x, int k)
 
 static int xoshiro128_initialize(const struct device *dev)
 {
-	dev = device_get_binding(DT_CHOSEN_ZEPHYR_ENTROPY_LABEL);
-	if (!dev) {
-		return -EINVAL;
+	dev = DEVICE_DT_GET(DT_CHOSEN(zephyr_entropy));
```

Comment:
Are you sure this should be a build error?

Reply:
Without this device, it appears that this rand source is not usable, so I think it should be. Will let maintainers judge here.

Reply:
> Will let maintainers judge here.

Can you `@`-mention the relevant maintainers, since this PR touches different subsystems?

Reply:
without an entropy source this module can't operate. It is fine for me to have a build error.